release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1102,6 +1102,62 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously, an alert for `OVNKubernetesNorthdInactive` would not fire in circumstances where it should fire. With this release, the issue is fixed so that the alert for `OVNKubernetesNorthdInactive` fires as expected. (link:https://issues.redhat.com/browse/OCPBUGS-33758[*OCPBUGS-33758*])
+
+* Previously, a missing route for the Kubernetes-OVN masquerade address in all pods in pods where the default route has been customized would cause each pod to be unable to connect to itself through a service for which it acts as backend. With this release, the missing route for Kubernetes-OVN masquerade address is added to pods so that the issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-36865[*OCPBUGS-36865*])
+
+* Previously, the `iptables-alerter` pod did not handle errors from `crictl`, which could cause the pod to incorrectly log events from `host-network` pods or cause pod restarts. With this release, the errors are handled correctly so that these issues no longer persist. (link:https://issues.redhat.com/browse/OCPBUGS-37713[*OCPBUGS-37713*])
+
+* Previously, if you created a hosted cluster by using a proxy for the purposes of making the cluster reach a control plane from a compute node, the compute node would be unavailable to the cluster. With this release, the proxy settings are updated for the node so that the node can use a proxy to successfully communicate with the control plane. (link:https://issues.redhat.com/browse/OCPBUGS-37786[*OCPBUGS-37786*])
+
+* Previously,  when a cluster failed to install on an on-premise platform with a configured load balancer, the `LoadBalancer` service's `LoadBalancerReady` condition received the `SyncLoadBalancerFailed` status. The status would trigger the generation of the `The kube-controller-manager logs might contain more details.` message. This message is wrong because the logs are stored in the `cloud-controller-manager` namespace of a project. With this release,  the `SyncLoadBalancerFailed` status now communicates the correct message: `The cloud-controller-manager logs may contain more details`. (link:https://issues.redhat.com/browse/OCPBUGS-31664[*OCPBUGS-31664*])
+
+* Previously, you could not control log levels for the internal component that selects IP addresses for cluster nodes. With this release, you can now enable debug log levels so that you can either increase or decrease log levels on-demand. To adjust log levels, you must create a config map manifest file with a configuration analogous to the following:
```

Comment:
```suggestion
* Previously, you could not control log levels for the internal component that selects IP addresses for cluster nodes. With this release, you can now enable debug log levels so that you can either increase or decrease log levels on-demand. To adjust log levels, you must create a config map manifest file with a configuration similar to the following:
```

Reply:
:heavy_check_mark: